src/framework/operations.hpp

```diff
@@ -940,6 +943,30 @@ inline Op make_qerror_loc(const reg_t &qubits, const std::string &label,
   return op;
 }
 
+// make new op by parameter binding
+inline Op make_parameter_bind(const Op &src, const uint_t iparam,
```

Comment:
I think this method name is `bind_parameters()`.

Reply:
changed name to `bind_parameter` because this function bind single parameter